product/builder-basics.mdx

```diff
@@ -10,6 +10,23 @@ The Visual Builder is composed of 4 sections:
 
 <VisualBuilderOverview />
 
+## Build & Interact Modes
+
+In the Component Toolbar you'll notice two options that dictate how you will engage with the elements on your Canvas; **Build** mode and **Interact** mode
+
+### Build
+
+Allows you to drag and drop components onto the canvas to become elemnts. Once on the canvas, stay in Build mode to drag your elements around, as well as select the elements to edit its properties. The majority of your creation sessions will happen in Build mode.
+
+### Interact
+
+Renders the canvas similarly to how your components will react on your live domain. You can use Interaction mode to engage with any interactive element (e.g. buttons, links, etc.) to see its behavior without utilizing [Publlishing](/product/publishing) or [Preview links](/product/publishing#previewing) and leaving the Makeswift Builder. You may also need to use Interaction mode to expose an editable section of an element, like an accordion for instance.
+
+<Frame>
+  ![Visual builder showing interaction mode for accordion
+  component](/images/builder/interact-mode.gif)
```

Comment:
@jamesqquick I would like to use an accordion component here as an example, but the venture template/default host doesn't have one. Should we just remove this all together or what do you think?

Reply:
We could just show a hover effect instead? On  a button or nav link?